lascli/parser/users.py

```diff
@@ -35,6 +42,8 @@ def create_users_parser(subparsers):
 
     create_user_parser = subparsers.add_parser('create')
     create_user_parser.add_argument('email')
+    create_user_parser.add_argument('--name', type=nullable, default=NotProvided)
+    create_user_parser.add_argument('--avatar', help='Path to avatar JPEG image.')
```

Comment:
isn't this also nullable?

Reply:
I don't think it makes sense to support nulling out a value that don't exist yet. There is nothing to null out